iree/hal/local/executable_library_benchmark.md

```diff
@@ -80,7 +80,7 @@ to many different formats/architectures.
 2. Translate the executable into the binary form consumed by the IREE loaders:
 
 ```
-iree-translate \
+iree-compile \
     -iree-mlir-to-hal-executable \
```

Comment:
Another place where `-iree-mlir-to-hal-executable` is used instead of the bytecode module output, similar to the HAL CTS CMake file. Which tool should this use?